server/internal/usecase/interactor/asset.go

```diff
@@ -78,6 +79,14 @@ func (i *Asset) Create(ctx context.Context, inp interfaces.CreateAssetParam, ope
 				return nil, err
 			}
 
+			th, err := thread.New().NewID().Workspace(prj.Workspace()).Comments([]*thread.Comment{}).Build()
```

Comment:
```suggestion
			th, err := thread.New().NewID().Workspace(prj.Workspace()).Build()
```

Note that a nil slice and an empty slice is almost same in Golang

Reply:
Done!